maintainers/maintainer-list.nix

```diff
@@ -2091,6 +2091,12 @@
     email = "christoph.senjak@googlemail.com";
     name = "Christoph-Simon Senjak";
   };
+  DavHau = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add davhau``.